conda_smithy/templates/circle.yml.tmpl

```diff
@@ -11,10 +11,7 @@ general:
 {%- if matrix -%}
 checkout:
   post:
-    # Checkout the merged PR for testing as CircleCI will just use the PR head otherwise.
-    - if [[ -n "${CIRCLE_PR_NUMBER}" ]]; then git fetch origin +refs/pull/$CIRCLE_PR_NUMBER/head:pr/$CIRCLE_PR_NUMBER/head ; fi
-    - if [[ -n "${CIRCLE_PR_NUMBER}" ]]; then git fetch origin +refs/pull/$CIRCLE_PR_NUMBER/merge:pr/$CIRCLE_PR_NUMBER/merge ; fi
-    - if [[ -n "${CIRCLE_PR_NUMBER}" ]]; then git checkout -qf pr/$CIRCLE_PR_NUMBER/merge ; fi
+    - ./ci_support/checkout_code.sh
```

Comment:
Have gone ahead with `checkout_merge_commit.sh`. Also have updated PR ( https://github.com/conda-forge/freeglut-feedstock/pull/6 ) to test this re-rendering and it still works.